packages/core/src/session/index.ts

```diff
@@ -142,7 +143,7 @@ export function storedSessions<Session>({
       return store.get(sessionId);
     },
     async start({ context, data }) {
-      const sessionId = uid(24);
+      const sessionId = randomBytes(24).toString('base64url'); // 192-bit
```

Comment:
I didn't want to change this, but it does seem arbitrary and I might up this to 256-bit in future